### ISSUE TITLE
Arb-editor on Open VSX version lags behind the one on VSC Marketplace

### ISSUE THREAD

**Issue by 0rzech** [google/arb-editor#77]
Arb-editor on Open VSX is two releases (still on version 0.0.13) behind the one on VSC Marketplace. Would it be possible to setup CI in a way that both versions get updated automatically on release?

**Comment by mosuem** [comment]
Good point. While at it, we should also set up CI to publish on the VS Code marketplace...

**Comment by 0rzech** [comment]
Both Open VSX and VSC Marketplace contain the same version of arb-editor now. Thank you very much!
